lib/send_sms.rb

```diff
@@ -0,0 +1,16 @@
+require 'twilio-ruby'
+
+time = Time.new + 3600 
+
+account_sid = "AC7c0dd1eab28593f5ec87bb03c409e40b"
```

Comment:
I'm sure you got the same email as me from github and twilio saying that it would stop this SID and token from working. Need to save them as environmental variables so that they are saved on your command line. This lets only you run them - or someone else can run this with their details as they will save their own details in their command line

Reply:
https://www.twilio.com/docs/usage/secure-credentials